cmd/cmd.go

```diff
@@ -82,6 +82,8 @@ func NewPackCommand(logger ConfigurableLogger) (*cobra.Command, error) {
 	rootCmd.AddCommand(commands.Version(logger, pack.Version))
 	rootCmd.AddCommand(commands.Report(logger, pack.Version))
 
+	rootCmd.AddCommand(commands.RegisterBuildpack(logger, cfg, &packClient))
```

Comment:
We would want to wrap this around experimental being enabled such as [`build --registry-url` flag](https://github.com/buildpacks/pack/blob/972ed1ef930df623bd8a0bdb9351519745b8f41f/internal/commands/create_builder.go#L65-L67).